wordpress-mobile/WordPress-Android
# Issue/update create icon

### Fix
Update the create icon with the new design per https://github.com/Automattic/gridicons/pull/256.

### Test
1. Go to ***Sites*** tab.
2. Notice new icon for floating action button.
3. Tap ***Blog Posts*** item in ***Publish*** section.
4. Notice new icon for floating action button.
